src/bnb_trading/main.py

```
@@ -0,0 +1,1476 @@
+        """
+        try:
+            # Зареждаме конфигурацията
+            self.config = toml.load(config_file)
+
+            # Инициализираме компонентите
+            self.data_fetcher = BNBDataFetcher(self.config["data"]["symbol"])
+            self.signal_generator = SignalGenerator(self.config)
+            self.validator = SignalValidator("data/results.csv")
+
+        except Exception as e:
+            logger.error(f"Грешка при инициализиране на системата: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)